packages/create-start/src/modules/core/template/tsconfig.json

```diff
@@ -0,0 +1,10 @@
+{
+  "compilerOptions": {
+    "jsx": "react-jsx",
+    "moduleResolution": "Bundler",
+    "module": "ESNext",
+    "target": "ES2022",
+    "skipLibCheck": true,
+    "strictNullChecks": true
```

Comment:
I would just use `strict: true` instead of `strictNullChecks`

Reply:
This is copied from the official docs https://tanstack.com/router/latest/docs/framework/react/start/getting-started